app/components/DetailsDropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Card, CardGroup, Icon } from '@trussworks/react-uswds'
+import { useState } from 'react'
+import type { ReactNode } from 'react'
+
+export default function DetailsDropdown({
+  summary,
+  children,
+  className,
+}: {
+  summary: ReactNode
+  children: ReactNode
+  className?: string
+}) {
+  const [showContent, setShowContent] = useState(false)
+
+  return (
+    <div className={className}>
+      <Button
+        className="display-flex flex-align-center usa-button usa-button--outline"
```

Comment:
The flex properties and the outline style may be specific to the version dropdown application.

```suggestion
        className="usa-button"
```